### ISSUE TITLE
Accessing Old Versions of the Building Footprint Data?

### ISSUE THREAD

**Issue by mtreg** [CityOfNewYork/nyc-geo-metadata#38]
Hi there - hope this is a reasonable place for this request/question, as I couldn't find an obvious best route for it: 
I understand based on the [Building Footprint Data on NYC Open Data](https://data.cityofnewyork.us/Housing-Development/Building-Footprints/nqwf-w8eh) that older versions of the NYC Building Data are available upon request.  I was hoping to track down the data as of May 2021 to align with some other data my team is working with.  Would appreciate any insights on how to access or request that. Feel free to follow up by email at michael.treglia@tnc.org

Best,
Mike Treglia
Lead Scientist, The Nature Conservancy - NY Cities Program

**Comment by mattyschell** [comment]
Hi again Mike.  I think your best option is to contact NYC Open Data using their contact page.  My understanding is that Open Data is able to provide past datasets upon request.  

https://opendata.cityofnewyork.us/engage/

Please report back here and let us know how it goes.  We can cook up other, less official options if NYC Open Data can't get you what you want.

**Comment by mtreg** [comment]
Thanks for the quick response! I submitted a request through NYC Open Data a couple of days ago - no response yet, but I'll let you know either way if I get one sometime next week.  

By dumb luck, happen to have a copy of the building footprint data from April 2021, which is probably close enough for all intents and purposes, so can just fall back to that  if I don't get anything else.

**Comment by mattyschell** [comment]
@mtreg please download this and let me know if it meets your needs.

https://nyc.maps.arcgis.com/home/item.html?id=eec6663a29d54779988d6e34d03beddb#overview

I checked in with  a colleague at NYC Open Data.  Though Open Data can usually find historic datasets, in  most cases they first refer requestors (you) to the source agency (me).  

Sorry about the runaround there, good to know how this is supposed to work!

**Comment by mtreg** [comment]
That's awesome - thanks so much @mattyschell! I just opened up the polygons and it looks like what I need.

And good to know that ultimately these kinds of requests should ultimately they end up referring folks to the source agencies for old versions of datasets. I actually had not heard back form the NYC Open Data Portal request that I made, so appreciate you checking in with the NYC Open Data Colleague.

For similar requests that might come up in the future, at least with OTI data, let me know if this is a good place to make the request, or if there's an email address or similar that would be preferable. (Not anticipating anything, but acknowledge that things come up)

Thanks again!
